packages/itwinui-css/src/input/input-flex-container.scss

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@mixin iui-input-flex-container {
+  --_iui-input-background-color: var(--iui-color-background);
+  --_iui-input-border-color: var(--iui-color-border-foreground);
+  --_iui-input-text-color: var(--iui-color-text);
+
+  @include iui-reset;
+  @include iui-focus($offset: -2px, $thickness: 2px);
+  width: 100%;
+  font-family: inherit;
+  font-size: var(--iui-font-size-1);
+  font-weight: var(--iui-font-weight-normal);
+  line-height: var(--iui-size-l);
+  border-radius: var(--iui-border-radius-1);
+  appearance: none;
```

Comment:
a lot of these styles, like `iui-reset`, `font-family`, `color`, `appearance`, `line-height`, etc are not needed on this element.

Reply:
input-flex-container css is reworked to have minimal css as needed.